contentrepo/settings/base.py

```diff
@@ -13,6 +13,7 @@
 DEFAULT_SECRET_KEY = "please-change-me"
 SECRET_KEY = os.environ.get("SECRET_KEY") or DEFAULT_SECRET_KEY
 ALLOWED_HOSTS = env.list("ALLOWED_HOSTS", default=["localhost"])
+WAGTAILDOCS_EXTENSIONS = ["doc", "docx", "xls", "xlsx", "ppt", "pptx", "pdf", "txt"]
```

Comment:
Maybe put a comment above the setting, saying that these are the extensions that whatsapp cloud API supports for documents (maybe with a link?), so that we know where we got this list from.